.github/workflows/docker.yml

```diff
@@ -0,0 +1,35 @@
+name: docker
+on:
+  push:
+    branches: [master]
```

Comment:
@mikiquantum should we instead do this only when tags are pushed? We don't necessarily want to push a Docker tag on every commit that is merged into master (think of a PR only changing CI-related files for examples).

Reply:
We've decided not to do this.